packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx

```diff
@@ -2,6 +2,7 @@ import styled from '@emotion/styled';
 
 const StyledDropdownMenu = styled.div<{
   disableBlur?: boolean;
+  disableBorder?: boolean;
```

Comment:
why do we disable border? Is Design inconsitent, if yes, it needs to be discussed with design, we should not allow this in the ui API unless this is a Design direction